.circleci/scripts/build_android_gradle.sh

```diff
@@ -10,6 +13,24 @@ export GRADLE_VERSION=4.10.3
 export GRADLE_HOME=/opt/gradle/gradle-$GRADLE_VERSION
 export GRADLE_PATH=$GRADLE_HOME/bin/gradle
 
+# touch gradle cache files to prevent expiration
```

Comment:
I'm not 100% sure about the purpose of this - moving it above everything else in case custom gradle build also needs it.

Reply:
@IvanKobzarev - could you please help review the gradle part of the PR?